src/terragraph-e2e/e2e/minion/BgpUtils.cpp

```diff
@@ -153,11 +148,9 @@ BgpUtils::fetchVtyshBgpStatus() {
     return bgpStatus;
   }
   std::string summaryCommand, advertisedRoutesFormat, receivedRoutesFormat;
-  if (versionOutput.value().rfind("Quagga", 0) == 0) {
-    summaryCommand = kVtyshQuaggaGetBgpSummaryCommand;
-    advertisedRoutesFormat = kVtyshQuaggaGetBgpAdvertisedRoutesFormat;
-    receivedRoutesFormat = kVtyshQuaggaGetBgpReceivedRoutesFormat;
-  } else /* if (versionOutput.value().rfind("FRRouting", 0) == 0) */ {
+ 
+  //FRROUTING
+   if (versionOutput.value().rfind("FRRouting", 0) == 0)  {
```

Comment:
Delete the whole "Get version" section too, since there is no more if/else. This code is wrong as is because you don't have an "else" block and it would crash if this condition doesn't trigger.

Reply:
Addressed the review comments.